scanners/flutter/flutter.go

```diff
@@ -214,58 +241,130 @@ func (scanner *Scanner) Options() (models.OptionModel, models.Warnings, error) {
 	flutterProjectLocationOption := models.NewOption(projectLocationInputTitle, projectLocationInputEnvKey)
 
 	for _, project := range scanner.projects {
-		if project.ProjectType == "app" {
-			projectPathOption := models.NewOption(ios.ProjectPathInputTitle, ios.ProjectPathInputEnvKey)
-			flutterProjectLocationOption.AddOption(project.path, projectPathOption)
-
-			for xcodeWorkspacePath, schemes := range project.xcodeProjectPaths {
-				schemeOption := models.NewOption(ios.SchemeInputTitle, ios.SchemeInputEnvKey)
-				projectPathOption.AddOption(xcodeWorkspacePath, schemeOption)
-
-				for _, scheme := range schemes {
-					exportMethodOption := models.NewOption(ios.IosExportMethodInputTitle, ios.ExportMethodInputEnvKey)
-					schemeOption.AddOption(scheme, exportMethodOption)
+		if project.hasTest {
+			flutterProjectHasTestOption := models.NewOption(testsInputTitle, "_")
```

Comment:
make sure an env with key `_` is not added to the `app/envs` in the generated bitrise.yml

Reply:
website is using the only env keys we set in the yml app envs section.

![screen shot 2019-01-21 at 3 12 54 pm](https://user-images.githubusercontent.com/1564933/51479651-2896a180-1d8f-11e9-8db9-0b76eca18f41.png):
![screen shot 2019-01-21 at 3 12 30 pm](https://user-images.githubusercontent.com/1564933/51479657-2fbdaf80-1d8f-11e9-9fb5-3482d56ff516.png)

with `_`:
![screen shot 2019-01-21 at 3 13 33 pm](https://user-images.githubusercontent.com/1564933/51479677-3f3cf880-1d8f-11e9-881d-df21f9ed8b1b.png):
![screen shot 2019-01-21 at 3 13 26 pm](https://user-images.githubusercontent.com/1564933/51479688-45cb7000-1d8f-11e9-9f3f-f4f57bc05111.png)